### ISSUE TITLE
Ignore LaTeX key-words

### ISSUE THREAD

**Issue by auzias** [yitzchak/linter-spell-latex#10]
Most* LaTeX keywords (and file-defined commands) are red-dashed-underligned.
* Is there a mean to ignore these?

> *: non-english ones such as `subsections` or `\ttfamily`, not the `section` or `chapter` ones.

Otherwise: **congrats !!** Great work!! Awesome package :+1:

**Comment by yitzchak** [comment]
The misspellings should show up as a warning which in linter is usually a yellow underline. Do you have the (spell-check)[https://atom.io/packages/spell-check] active?

**Comment by auzias** [comment]
Here is the list of the packages I use:
**TL;DR**: no, I do not have `spell-check` package active.

```
git-time-machine@1.5.3
intentions@1.1.2
language-latex@0.6.1
latexer@0.3.0
linter@1.11.16
linter-spell@0.10.3
linter-spell-latex@0.8.0
minimap@4.24.7
```

**Comment by yitzchak** [comment]
If the red-underline is a spell error from linter-spell it should respond to `alt-enter` for the intentions package. Can you do a screen shot?

BTW, spell-check is core package so it will be the "Core Packages" list.

**Comment by auzias** [comment]
You're right. No need for screenshot: when the `spell-check` (that _was_ enabled, indeed) is disabled there isn't red underlined words anymore.

**Comment by yitzchak** [comment]
Great. Seems like this could be a common problem, so I am going to tag this as enhancement. I will eventually add a check for spell-check being active and give a warning to the user in that case.

Thanks for reporting!

**Comment by auzias** [comment]
You're welcome!

You may wish to add this [GH issue link](https://github.com/atom/spell-check/issues/126) to your warning and suggest users to request solving this bug or even do a PR.

Great work again!

**Comment by yitzchak** [comment]
Fixed by yitzchak/linter-spell/#ed76f6326a6cff21614930bddbdc1a704e65707b
